console/console-init/ui/mock-console-server/mock-console-server.js

```diff
@@ -1058,34 +1188,50 @@ function createAddress(addr, addressSpaceName) {
       type: addr.spec.type,
       topic: addr.spec.topic
     },
-    status: null,
+    status: {
+      phase: "",
+      messages: []
+    }
```

Comment:
@k-wall I made a small change in the mock server as it is not matching with exact behaviour of real backend.